vgsshow/src/main/java/com/verygoodsecurity/vgsshow/widget/core/VGSView.kt

```diff
@@ -139,6 +143,39 @@ abstract class VGSView<T : View> @JvmOverloads internal constructor(
      */
     fun getContentPath(): String = contentPath ?: ""
 
+    /**
+     * Describes how to determinate if this input field is important for accessibility.
+     *
+     * @param mode Accessibility mode.
+     *
+     * @see View.IMPORTANT_FOR_ACCESSIBILITY_YES
+     * @see View.IMPORTANT_FOR_ACCESSIBILITY_NO
+     * @see View.IMPORTANT_FOR_ACCESSIBILITY_NO_HIDE_DESCENDANTS
+     * @see View.IMPORTANT_FOR_ACCESSIBILITY_AUTO
+     *
+     * @attr [R.styleable.VGSView_importantForAccessibility]
+     */
+    override fun setImportantForAccessibility(mode: Int) {
+        // Update the value in the text view itself
+        view.importantForAccessibility = mode
+        // Disable accessibility in the container view
+        super.setImportantForAccessibility(View.IMPORTANT_FOR_ACCESSIBILITY_NO)
```

Comment:
@DonaldRG What is the difference between show and collect here? In collect we pass same mode for child and parent views. https://github.com/verygoodsecurity/vgs-collect-android/pull/555/files#diff-b483c877a6fc7180b2140f0b82fae5a155e45a97faa9290d892be6eee71d7956R768

Reply:
That's a good catch, I will update the collect SDK since it is better to only leave accessibility available in the view itself, to avoid the talk back potentially read the same description twice.